---
id: 647e22958acb466c97ccbef4
title: 'Керований проєкт: налагодьте та обробіть винятки в консольному застосунку C#, використовуючи Visual Studio Code'
challengeType: 19
dashedName: guided-project-debug-and-handle-exceptions-in-a-c-sharp-console-application-using-visual-studio-code
---

# --description--

Це завдання буде частково виконано на платформі Microsoft Learn. Дотримуйтесь цих інструкцій, щоб виконати завдання:

1. Перейдіть до <a href="https://learn.microsoft.com/training/modules/guided-project-debug-handle-exceptions-c-sharp-console-application/" target="_blank" rel="noreferrer">https://learn.microsoft.com/training/modules/guided-project-debug-handle-exceptions-c-sharp-console-application/</a> та виконайте всі завдання модуля «Керований проєкт: налагодьте та обробіть винятки в консольному застосунку C#, використовуючи Visual Studio Code». Це **обов’язково**, щоб отримати трофей з «Налагодження консольних застосунків C#» на Microsoft Learn та мати змогу скласти сертифікаційний екзамен.
1. Як тільки завершите, поверніться та дайте правильну відповідь на запитання нижче.

# --assignment--

Виконайте керований проєкт <a href="https://learn.microsoft.com/training/modules/guided-project-debug-handle-exceptions-c-sharp-console-application/" target="_blank" rel="noreferrer">«Налагодьте та обробіть винятки в консольному застосунку C#, використовуючи Visual Studio Code»</a> на Microsoft Learn. Потім дайте відповідь на запитання нижче.

# --questions--

## --text--

When is an `ArgumentOutOfRangeException` exception thrown?

## --answers--

An `ArgumentOutOfRangeException` exception is thrown when an attempt is made to index an array outside the bounds of the array.

---

An `ArgumentOutOfRangeException` exception is thrown when the value of an argument is outside the allowable range of values as defined by the method.

---

An `ArgumentOutOfRangeException` exception is thrown when an attempt is made to store a value of one type in an array of another type.

## --video-solution--

2
